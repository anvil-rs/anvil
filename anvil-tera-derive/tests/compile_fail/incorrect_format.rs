// This file tests the error when a template has an incorrectly formatted template attribute
// Expected error: "Expected #[template(...)] attribute list format, e.g., #[template(path = \"...\")]"

use anvil_tera_derive::Template;
use serde::Serialize;

#[derive(Serialize, Template)]
#[template = "test.txt"]  // Using incorrect format
struct IncorrectFormatTemplate {
    value: String,
}

fn main() {
    // This will never run, we're only testing the compile-time error
}
