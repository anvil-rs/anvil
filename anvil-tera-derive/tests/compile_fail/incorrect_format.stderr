error: Expected #[template(...)] attribute list format, e.g., #[template(path = "...")]
 --> tests/compile_fail/incorrect_format.rs
  |
  | #[template = "test.txt"]  // Using incorrect format
  | ^^^^^^^^^^^^^^^^^^^^^^^^
